proposals/lip-0060.md

```diff
@@ -41,17 +41,21 @@ Another distinctive specification of the genesis block is its processing. The bl
 
```

Comment:
For consistency we could also just name this section "Static validation"

Reply:
Also, in the previous paragraph we should amend "In particular the processing of the genesis block happens in **four** steps described in the sections below." --> "In particular the processing of the genesis block happens in **five** steps described in the sections below." (not sure if 'block storage should be included, if so, it should be 6)

Reply:
- I changed all occurrences of "Static validation of the genesis block" to "Static validation".
- Changed it from four to five steps.

Reply:
I would add the 'block storage' step to the description (you can copy/paste it from LIP 0055), since I put it in the diagram, and update from 'five' to 'six'.

Reply:
I added the block storage to the description in LIP 60.